marimo/_config/config.py

```diff
@@ -91,14 +91,28 @@ class RuntimeConfig(TypedDict):
     - `on_cell_change`: if `lazy`, cells will be marked stale when their
       ancestors run but won't autorun; if `autorun`, cells will automatically
       run when their ancestors run.
-    - `execution_type`: if `relaxed`, marimo will not clone cell declarations;
-      if `strict` marimo will clone cell declarations by default, avoiding
-      hidden potential state build up.
+    - `output_max_byes`:  Byte limits on outputs. Limits exist for two reasons:
```

Comment:
I copied this over from _messaging/streams.py.  It's probably a bit verbose for a GUI message.

Reply:
i think we lost `execution_type` docs here. i think we can keep this (but don't need to add it below). its a hidden config that still might be in use

Reply:
That probably should have just been a comment in the PR because it distracts from this PR.  I removed the docs because I didn't see the config setting, I'll add it back in my PR.

Reply:
Fixed.